proco/locations/models.py

```diff
@@ -65,6 +68,28 @@ class Meta:
     def __str__(self):
         return f'{self.name}'
 
+    def _calculate_batch_avg_distance_school(self, points):
+        earth_radius = 6371.0088
+        dist = DistanceMetric.get_metric('haversine')
+        distances = dist.pairwise(np.radians(points))
+        indexes = np.tril_indices(n=distances.shape[0], k=-1, m=distances.shape[1])
+        return earth_radius * np.mean(distances[indexes])
+
+    def calculate_avg_distance_school(self):
+        schools_count = self.schools.count()
+        schools_point = self.schools.annotate(
+            lon=models.Func(models.F('geopoint'), function='ST_X', output_field=models.FloatField()),
+            lat=models.Func(models.F('geopoint'), function='ST_Y', output_field=models.FloatField()),
+        ).values_list('lat', 'lon')
+
+        if schools_count < 2:
+            return None
```

Comment:
avg_distance_school is not nullable. please change field or returned value

Reply:
Changed field